core/org.openjdk.jmc.flightrecorder/src/main/java/org/openjdk/jmc/flightrecorder/jdk/JdkAttributes.java

```diff
@@ -136,7 +136,6 @@ public IQuantity getMember(U i) {
 					};
 				}
 			});
-
```

Comment:
Mainly out of curiosity, but I noticed that there are still extra lines scattered through this file, but I couldn't figure out a pattern. Is there a purpose behind the lines removed here and the ones that still remain?

Reply:
Well, some of them seemed to _maybe_ indicate some sort of grouping so I left them. I should probably just remove them all. ;)